src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -396,21 +468,76 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
                 position: 'relative',
               }}
             >
-              <SemiTransparentHeader
-                analysisName={analysisState.analysis?.displayName}
-                entityDisplayName={outputEntity?.displayNamePlural || 'Samples'}
-                filterList={<FilterChipListForHeader />}
-                isExpanded={mapHeaderIsExpanded}
-                logoProps={props.logoProps}
-                onAnalysisNameEdit={analysisState.setName}
-                onToggleExpand={() => setMapHeaderIsExpanded((c) => !c)}
-                studyName={studyRecord.displayName}
-                totalEntityCount={outputEntityTotalCount}
-                totalEntityInSubsetCount={totalEntityCount}
-                visibleEntityCount={
-                  totalVisibleWithOverlayEntityCount ?? totalVisibleEntityCount
-                }
-              />
+              <div
+                style={{
+                  // Make a div that completely fills its parent. Have it
+                  // layout its children with flexbox.
+                  display: 'flex',
+                  flexDirection: 'column',
+                  height: '100%',
+                  width: '100%',
+                  // Attach this div container to it's parent.
+                  position: 'absolute',
+                  // Remember that just about everything in the DOM is box.
+                  // This div is sitting on top of the map. By disabling
+                  // pointer events we are saying: hey, div, become porous.
+                  // If a user clicks you, don't capture it, but let it go
+                  // to the map you're covering.
+                  pointerEvents: 'none',
```

Comment:
![image](https://user-images.githubusercontent.com/24446573/223844504-c63695f8-358b-4b68-8abb-fdef9eb7bb9f.png)

By default everything in the DOM is set to `pointer-events: auto`. As you can seem, my layout is superimposed atop the map. That means it'll catch all the clicks and our poor map won't have any. If I set the parent container to `pointer-events: none` then I'm telling it to be a ghost. You can see it, but you can't touch it.

This is a problem because I _do_ want clicks on the header and side navigation. So I set `pointer-events: auto` onto the children to restore that property _only for those elements_. The parent is still `pointer-events: none` 👻